### PR TITLE
Remove jQuery `.attr` from the repository settings

### PR DESCRIPTION
**Pull request by yardenshoham**
- Switched from jQuery `.attr` to plain javascript `getAttribute` and `setAttribute`
- Tested the collaborator access mode change, team search box, and branch protection form. They all work as before

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: web_src/js/features/repo-settings.js [hunk 1]
```diff
@@ -13,17 +13,17 @@ export function initRepoSettingsCollaboration() {
     const $text = $dropdown.find('> .text');
     $dropdown.dropdown({
       async action(_text, value) {
-        const lastValue = $dropdown.attr('data-last-value');
+        const lastValue = e.getAttribute('data-last-value');
```
Comment (silverwind) [inline web_src/js/features/repo-settings.js]: These are all wrong. You meant `e.target` instead of `e`. I doubt this was properly tested.

Comment (yardenshoham) [reply web_src/js/features/repo-settings.js]: `e` is an HTMLElement, not an event

Comment (silverwind) [reply web_src/js/features/repo-settings.js]: Ah ok. can you please rename to `el`?

Comment (yardenshoham) [reply web_src/js/features/repo-settings.js]: Done

**Review by silverwind — APPROVED** [review]

**Pull request merged**